docs-api/frontity-packages/features-packages/wp-source.md

```diff
@@ -233,17 +234,48 @@ module.exports = {
 
 and then you visit a URL \(or use `actions.source.fetch`\), the query part of the HTTP call to the REST API will be `per_page=5&type[]=post&type[]=page`.
 
+#### `state.source.auth`
+
+This option is a property which can hold authentication information. This could be a JWT token or a Basic Authentication string, or another type of authentication.
+
+Values can be passed to `state.source.auth` in a variety ways, e.g. via `frontity.settings.js` or by setting it in the way that any other piece of frontity state is.
+
+```js
+// frontity.settings.js
+const state = {
+  source: {
+    auth: "Basic YWFhOmJiYg",
+  },
+};
+```
+
+Additionally the value of `state.source.auth` can be set via a **query string**. If a `frontity_source_auth` param is present in the URL `state.source.auth` will use its value. Values passed in this way will be removed from the `initialLink` URL and added to `state.frontity.options`.
```

Comment:
Should we add another sentence clarifying that `frontity_source_auth` is a Frontity Query Option?